apps/labrinth/src/queue/payouts.rs

```diff
@@ -651,8 +651,8 @@ impl PayoutsQueue {
     ) -> Result<Option<AccountBalance>, ApiError> {
         #[derive(Deserialize)]
         struct FundingSourceMeta {
-            available_cents: u64,
-            pending_cents: u64,
+            available_cents: Option<u64>,
+            pending_cents: Option<u64>,
```

Comment:
You need to add a `#[serde(default)]` annotation to the struct for these fields to default to `None` when not present.

Reply:
That's interesting. I remember having to add it years ago in at least one occasion for that to work. [And it's documented as such by serde](https://serde.rs/attr-default.html). Maybe it depends on the deserializer implementation. I suppose there is a point in annotating it explicitly so no relevant behavior is left implicit to undocumented details, but as it works like this, there is no hard need to change it either.